lib/modules/manager/gradle/parser.spec.ts

```diff
@@ -59,6 +59,105 @@ describe('modules/manager/gradle/parser', () => {
       });
     });
 
+    describe('Groovy: multi var assignments', () => {
+      it('simple map', () => {
+        const input = `
+        ext {
+          versions = [
+            spotbugs_annotations  : '4.5.3',
+            core                  : '1.7.0',
+          ]
+
+          ignored = [ 'asdf' ]
+
+          libraries = [:]
+          libraries += [
+            guava: "com.google.guava:guava:31.1-jre",
+            detekt: '1.18.1',
+          ]
+        }
+        `;
+
+        const output = {
+          'versions.spotbugs_annotations': '4.5.3',
+          'versions.core': '1.7.0',
+          'libraries.guava': 'com.google.guava:guava:31.1-jre',
+          'libraries.detekt': '1.18.1',
+        };
+
+        const { vars } = parseGradle(input);
+        for (const [key, value] of Object.entries(output)) {
+          expect(vars).toContainKey(key);
+          expect(vars[key]).toMatchObject({ key, value });
```

Comment:
can't this be done without a loops? can't we use a simple single object match?

asking because of performance. more expectations are probably slower.